content/references/rippled-api/ledger-data-formats/ledger-object-types/signerlist.md

```diff
@@ -42,33 +42,47 @@ The `SignerList` object type represents a list of parties that, as a group, are
-| `SignerWeight`    | Number    | UInt16        | The weight of a signature from this signer. A multi-signature is only valid if the sum weight of the signatures provided meets or exceeds the SignerList's `SignerQuorum` value. |
+| Name           | JSON Type | Internal Type | Description                     |
+|:---------------|:----------|:--------------|:--------------------------------|
+| `Account`      | String    | AccountID     | An XRP Ledger address whose signature contributes to the multi-signature. It does not need to be a funded address in the ledger. |
+| `SignerWeight` | Number    | UInt16        | The weight of a signature from this signer. A multi-signature is only valid if the sum weight of the signatures provided meets or exceeds the SignerList's `SignerQuorum` value. |
 
 When processing a multi-signed transaction, the server dereferences the `Account` values with respect to the ledger at the time of transaction execution. If the address _does not_ correspond to a funded [AccountRoot object](accountroot.html), then only the master secret associated with that address can be used to produce a valid signature. If the account _does_ exist in the ledger, then it depends on the state of that account. If the account has a Regular Key configured, the Regular Key can be used. The account's master key can only be used if it is not disabled. A multi-signature cannot be used as part of another multi-signature.
 
+## {{currentpage.name}} Flag
+
+_Requires the [MultiSignReserve Amendment](known-amendments.html#multisignreserve)._
+
+SignerList objects can have the following flag value:
+
+| Flag Name        | Hex Value  | Decimal Value | Description                    |
+|:-----------------|:-----------|:--------------|:-------------------------------|
+| lsfOneOwnerCount | 0x00010000 | 65536         | This flag is set by the [SignerListSet transaction](signerlistset.html) when you create a new SignerList, or replace or remove a SignerList that was created before the MultiSignReserve amendment was enabled. Setting this flag enables the XRP Ledger to lower the [`OwnerCount`](accountroot.html#accountroot-fields) and [owner reserve](reserves.html#owner-reserves) for a SignerList as provided by the MultiSignReserve amendment. For example, when you create a SignerList after the MultiSignReserve amendment is enabled, this flag is set, your `OwnerCount` is increased by 1, and your owner reserve is increased by 5 XRP. When you replace a SignerList that was created before the MultiSignReserve amendment was enabled, this flag is set, your `OwnerCount` and owner reserve for the replaced SignerList are removed, your `OwnerCount` is increased by 1, and your owner reserve is increased by 5 XRP. When you remove a SignerList that was created before the MultiSignReserve amendment was enabled, this flag is set and your `OwnerCount` and owner reserve for the SignerList you removed are removed. |
```

Comment:
This is pretty wordy. I would consider simplifying it to:

> If this flag is enabled, this SignerList counts as one item for purposes of the [owner reserve]. Otherwise, this list counts as N+1 items, where N is the number of signers it contains. This flag is automatically enabled if you add or update a signer list after the [MultiSignReserve amendment] is enabled.

Reply:
Revised description looks good, but I think it's "N+2 items", since the owner count for a signer list is currently 3 to 10?

Reply:
@mDuo13 - any thoughts on @scottschurr's suggested adjustment to the item count? 

The existing doc says that a SignerList counts at 2 objects. So, I think N+2 is correct -- where 2 is the item count for the SignerList and N is the number of signers?

Reply:
Yes, that was a typo in my comment. Sorry.